photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditorView.java

```diff
@@ -35,7 +35,7 @@
 
     private FilterImageView mImgSource;
     private BrushDrawingView mBrushDrawingView;
-    private ImageFilterView mImageFilterView;
+    private ZedgeImageFilterView mImageFilterView;
```

Comment:
Why do we need this? ZedgeImageFilterView extends ImageFilterView.

Reply:
Because I added new methods to it that are used here.